BlockTerrain/ChunkMeshGenerator.cs

```diff
@@ -50,9 +50,9 @@ private Mesh GenerateMesh(Chunk updatedChunk)
             }
         }
 
-        mesh.vertices = generatedVertices.ToArray();
-        mesh.colors = generatedColors.ToArray();
-        mesh.triangles = generatedTriangles.ToArray();
+        mesh.SetVertices(generatedVertices);
+        mesh.SetColors(generatedColors);
+        mesh.SetTriangles(generatedTriangles);
```

Comment:
I can't find a version of [`SetTriangles`](https://docs.unity3d.com/2018.2/Documentation/ScriptReference/Mesh.SetTriangles.html) that takes a single argument. Updating this line to `mesh.SetTriangles(generatedTriangles, 0);` does work though.